dev-util/pkgconf/pkgconf-1.3.90.recipe

```diff
@@ -0,0 +1,121 @@
+SUMMARY="A lightweight pkg-config replacement"
+DESCRIPTION="A program which helps to configure compiler and linker flags for \
+development libraries. It is similar to pkg-config from freedesktop.org."
+HOMEPAGE="http://pkgconf.org/"
+COPYRIGHT="2011-2017 pkgconf authors"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="https://distfiles.dereferenced.org/pkgconf/pkgconf-$portVersion.tar.xz"
+CHECKSUM_SHA256="112db7f957816901b5fe448715c0a9e38f171d4794c44075b0734a0e7fcfba36"
+PATCHES="pkgconf-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix="$secondaryArchSuffix"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=""
+fi
+
+libVersion="2.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	pkgconf$secondaryArchSuffix = $portVersion
+	cmd:pkgconf$secondaryArchSuffix = $portVersion
+	lib:libpkgconf$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	pkgconf${secondaryArchSuffix}_devel = $portVersion
+	devel:libpkgconf$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	pkgconf$secondaryArchSuffix == $portVersion base
+	"
+
+SUMMARY_pkgconfig="$SUMMARY (pkg-config compat symlink)"
+DESCRIPTION_pkgconfig="$DESCRIPTION (pkg-config compatibility symlink)"
```

Comment:
I would either not define `DESCRIPTION_pkgconfig` at all or, if you prefer, keep it but with this change:
```diff
-DESCRIPTION_pkgconfig="$DESCRIPTION (pkg-config compatibility symlink)"
+DESCRIPTION_pkgconfig="$DESCRIPTION
+(pkg-config compatibility symlink)"
```

Reply:
Well, after all, the `DESCRIPTION_pkgconfig` is OK as it is now. I'll delete my previous comment.